### PR TITLE
Post Carousel: Add Keyboard Navigation

### PR DESCRIPTION
**Pull request by AlexGStapleton**
Resolve https://github.com/siteorigin/so-widgets-bundle/issues/1108

- The first item is the default tab index.
- Upon tabbing to tab index slide, use left and right keyboard arrows to navigate to the next/previous item
- The tab index is the currently active item
- Add keyboard navigation for post carousel arrows by pressing Enter or Space. Will navigate entire carousel slides rather than item by item.

`build:dev` is required due to LESS changes.

### CODE REVIEW

**Comment by Misplon** [comment]
Thanks, mate.

* When using enter to scroll I can click the enter button six times I think before focus moves away from the carousel. Are you able to replicate?

**Comment by AlexGStapleton** [comment]
How does that look?

I think we should add a setting in a future PR that allows you to change the navigation background/icon color and hover colors. The current hover color is honestly really hard to see.

**Comment by Misplon** [comment]
> I think we should add a setting in a future PR that allows you to change the navigation background/icon color and hover colors. The current hover color is honestly really hard to see.

For sure, please, proceed.

File: widgets/post-carousel/js/carousel.js [hunk 1]
```diff
@@ -106,7 +110,51 @@ jQuery( function ( $ ) {
...
+
+		// Keyboard Navigation of carousel items.
+		$( document ).on( 'keyup', '.sow-carousel-item', function( e ) {
+			// Ensure left/right key was pressed
+			if ( e.keyCode != 37 && e.keyCode != 39 ) {
+				return;
+			}
+
+			var $wrapper =  $( this ).parents( '.sow-carousel-wrapper' ),
+			$items = $wrapper.find( '.sow-carousel-items' ),
```
Comment (gregpriday) [inline widgets/post-carousel/js/carousel.js:132]: For clarity, I always prefer it if these are indented one level. Not sure what the WordPress style guide says here though.

Comment (AlexGStapleton) [reply widgets/post-carousel/js/carousel.js]: The Style guide actually recommends the spacing also. 😞

**Pull request merged**